### PR TITLE
[MFTF] Refactoring of AdminMassOrdersCancelCompleteAndClosedTest

### LINKED ISSUE #31129: [Issue] [MFTF] Refactoring of AdminMassOrdersCancelCompleteAndClosedTest
This issue is automatically created based on existing pull request: magento/magento2#31123: [MFTF] Refactoring of AdminMassOrdersCancelCompleteAndClosedTest

---------
### Description (*)
A new test has been created according to the best practices followed by MFTF.
The old "AdminMassOrdersCancelCompleteAndClosedTest" has been deprecated.

### Manual testing scenarios (*)
1. Create two orders
2. Invoice and ship the first order
3. Invoice and refund the second order
4. Go to the Orders Grid
5. Try to cancel both orders via mass update
6. Verify the result

### PR DESCRIPTION
**Pull request by AnnaAPak**
### Description (*)
A new test has been created according to the best practices followed by MFTF.
The old "AdminMassOrdersCancelCompleteAndClosedTest" has been deprecated.

### Manual testing scenarios (*)
1. Create two orders
2. Invoice and ship the first order
3. Invoice and refund the second order
4. Go to the Orders Grid
5. Try to cancel both orders via mass update
6. Verify the result

### Resolved issues:
1. [x] resolves magento/magento2#31129: [MFTF] Refactoring of AdminMassOrdersCancelCompleteAndClosedTest

### CODE REVIEW

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

File: app/code/Magento/Sales/Test/Mftf/Test/AdminMassOrdersCancelCompleteAndClosedAPITest.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminMassOrdersCancelCompleteAndClosedAPITest">
```
Comment (rogyar) [inline app/code/Magento/Sales/Test/Mftf/Test/AdminMassOrdersCancelCompleteAndClosedAPITest.xml]: I would suggest removing the API keyword from the test name. Despite the fact we create entities via API, we test the admin UI behavior eventually (not the API). For API we have a different type of tests. 

```suggestion
    <test name="AdminMassOrdersCancelCompleteAndClosedAPITest">
```

**Review by rogyar — CHANGES_REQUESTED** [review]
Hi @AnnaAPak. Awesome piece of work! 
Please, check my suggestion regarding the naming.

**Comment by sidolov** [comment]
@magento create issue

**Comment by engcom-Charlie** [comment]
Hello @AnnaAPak could you please update your PR accordingly to https://github.com/magento/magento2/pull/31123#discussion_r534155321?
Thank you.

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

File: app/code/Magento/Sales/Test/Mftf/Test/AdminMassOrdersCancelCompleteAndClosedAPITest.xml (hunk 1 above)
Comment (AnnaAPak) [reply app/code/Magento/Sales/Test/Mftf/Test/AdminMassOrdersCancelCompleteAndClosedAPITest.xml]: Hi @rogyar, I have renamed the test. Thank you!

**Comment by rogyar** [comment]
@magento run Functional Tests EE, Functional Tests B2B

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Review by rogyar — APPROVED** [review]
Failing tests for StorePickup are not related to the current refactoring

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-8538](https://jira.corp.magento.com/browse/ENGCOM-8538) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
**Note:** QA not applicable

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Pull request merged**